modules/sandboxed-containers-create-kata-config-resource-cli.adoc

```diff
@@ -24,6 +24,20 @@ Kata is installed on all worker nodes by default. If you want to install `kata`
 * You have access to the cluster as a user with the `cluster-admin` role.
 * You have installed the {sandboxed-containers-operator}.
 
+[IMPORTANT]
+====
+Creating the KataConfig CR automatically reboots the worker nodes. The reboot can take anywhere between 10 minutes and an hour, depending on the size of the deployment and the number of worker nodes.
```

Comment:
The reboot depends on the number of worker nodes and the time it takes the nodes to reboot in your environment.  It can take as few as 10 minutes or longer than an hour.

ex: I've seen a node with all the diagnostics turned on take 30 minutes to reboot.  If you have 3 nodes, that's over 1.5 hours.